jenkins/cicd/cortx-ha-dep.sh

```diff
@@ -61,7 +61,7 @@ if [ "$DEV" == "false" ]; then
     then
         req_file=${BASE_DIR}/jenkins/pyinstaller/v1/requirements.txt
     else
-        req_file=${BASE_DIR}/jenkins/pyinstaller/v2/requirements.txt
+        req_file=${BASE_DIR}/requirements.txt
```

Comment:
We had decide a directory structure for such files. Let's keep the directory structure intact. Please revert the path.

Reply:
Reverted.